examples/basic-top-down/components/player-controller.js

```diff
@@ -3,6 +3,24 @@ var PlayerController = pc.createScript('playerController');
 PlayerController.attributes.add('speed', { type: 'number' });
 PlayerController.attributes.add('jumpForce', { type: 'number', default: 1500 });
 
+PlayerController.prototype.initialize = function() {
+    const userId = this.entity.script.networkEntity.owner;
+    const user = this.app.room.users.get(userId);
+
+    if (!user)
+        return;
+
+    user.on('player:input', this.setInput, this);
+
+    this.app.room.on(`leave:${userId}`, () => {
```

Comment:
Add issue to add events for the User class.
Also User should be unique per room.